README-lede_generic.md

```diff
@@ -15,3 +15,32 @@ The agent has its own version that is generated if it is built separately
 from the firmware (only happens during development, it is based on time).
 For the release builds the agent version and the firmware version are the
 same.
+
+## Platforms
+
+Following platforms are supported so far.
+ - TPLink Archer C7 v2/v4/v5
+ - GL - Inet B1300
+ - GL - Inet AR300M
+ - Linksys WRT1900CS
+ - TPLink - WDR3600 V1
+
+## Build
+
+ LEDE code with unum-specifc build config files is here:
+ https://github.com/violetatrium/lede
+
+ Make sure unum-sdk is accessisble from lede as ../unum-sdk.
+
+ Build can be triggered with the following command from lede directory:
+ ./build -d -e <platform name>
```

Comment:
No need to build with '-e' anymore, the build script now builds with multiple CPUs then switches to -j1 if error is detected.

Reply:
Removed option '-e'.